*** ./smali/android/preference/PreferenceActivity.smali	2014-05-24 22:39:14.568660488 +0800
--- /home/xiangxin/patchrom/miuiv5/u8500_stock/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceActivity.smali	2014-05-24 22:37:48.760663844 +0800
***************
*** 2045,2050 ****
--- 2045,2052 ----
  
      if-lez v17, :cond_e
  
+     invoke-static/range {p0 .. p0}, Landroid/preference/PreferenceActivity$Injector;->setListViewPadding(Landroid/preference/PreferenceActivity;)V
+ 
      new-instance v17, Landroid/preference/PreferenceActivity$HeaderAdapter;
  
      move-object/from16 v0, p0
