*** ./smali/com/android/internal/statusbar/IStatusBar.smali	2014-05-24 22:40:15.812658093 +0800
--- /home/xiangxin/patchrom/miuiv5/u8500_stock/temp/new_smali/framework.jar.out/./smali/com/android/internal/statusbar/IStatusBar.smali	2014-05-24 22:38:40.328661827 +0800
***************
*** 103,108 ****
--- 103,116 ----
      .end annotation
  .end method
  
+ .method public abstract setStatus(ILjava/lang/String;Landroid/os/Bundle;)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setSystemUiVisibility(II)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
