*** ./smali/android/view/LayoutInflater.smali	2014-05-24 22:39:05.168660856 +0800
--- /home/xiangxin/patchrom/miuiv5/u8500_stock/temp/new_smali/framework.jar.out/./smali/android/view/LayoutInflater.smali	2014-05-24 22:37:38.992664226 +0800
***************
*** 1503,1508 ****
--- 1503,1516 ----
  
      move-result-object v1
  
+     invoke-static {v1, p1}, Landroid/view/LayoutInflater$Injector;->getResourceId(Landroid/content/Context;I)I
+ 
+     move-result p1
+ 
+     invoke-virtual {p0}, Landroid/view/LayoutInflater;->getContext()Landroid/content/Context;
+ 
+     move-result-object v1
+ 
      invoke-virtual {v1}, Landroid/content/Context;->getResources()Landroid/content/res/Resources;
  
      move-result-object v1
